compiler/optimizer/BitVectorAnalysis.cpp

```diff
@@ -1249,18 +1240,23 @@ template<class Container>bool TR_ForwardDFSetAnalysis<Container *>::analyzeNodeI
            bitVector = analysisInfo->getContainer(analysisInfo->_regularKillSetInfo, node->getNumber());
            if (bitVector)
               *this->_temp -= *bitVector;
-
-           bitVector = analysisInfo->getContainer(analysisInfo->_exceptionKillSetInfo, node->getNumber());
-           if (bitVector)
-              *this->_temp -= *bitVector;
+           else
```

Comment:
This is a semantic difference - previously we would have alwyas done the subtraction form the exceptionKillSetInfo if there was such a BV - now we will only do it there is no regularKillSetInfo. Is that right? I would have thought updating the exception sets would be required - could you add some reasoning for this change since it isn't in the commit message or in the PR message (which suggests this purely clean up).

Reply:
The commit message did mention this change:

```Clean up bit vector analysis.

This commit fixes some indentation issues and deletes unused code
in both bit vector analysis and backward bit vector analysis.
It also skips checking the same block as regular and
exception successor in bit vector analysis.
```

The rationale is that we have an IL rule in the compiler that disallows the same block to be the target of both a regular CFG edge and an exception CFG edge.

Reply:
ok thanks